ssl/ssl_cert.c

```diff
@@ -782,6 +783,79 @@ int SSL_add_dir_cert_subjects_to_stack(STACK_OF(X509_NAME) *stack,
     return ret;
 }
 
+/**
+ * Add a file of certs to a stack.
```

Comment:
A file of certs? But we're talking about URIs ... might not be a file?

Reply:
Ah, should be "container"

Reply:
Fixed